docs/installation/azuread.asciidoc

```diff
@@ -0,0 +1,83 @@
+    1. Under 'Grant', select 'Grant access' and check 'Require multi-factor authentication' and any other settings your organization requires.
+    1. At the bottom, make sure 'Enable policy' is set to 'On' and save your policy
+
+==== Disabling MFA without Azure AD premium
+
+WARNING: This will disable common recommended settings from Microsoft. Using Azure AD premium is the correct way to perform this. This option is only suggested for testing or when its impossible to have access to Azure AD premium.
+
+ 1. Open the "Azure Active Directory" in your Azure portal
+ 1. Under 'Manage', open 'Properties'
+  1. Click 'Manage Security defaults'
+  1. Disable the toggle 'Enable Security defaults' and save
+
+=== Configuring PacketFence
+
+ 1. Under 'Configuration->Policies and Access Control->Authentication Sources', create a new 'Azure Active Directory' internal source
+  1. Client ID: the 'Client ID' that was displayed while configuring the 'PacketFence' app inside Azure
+  1. Client Secret: the secret you created inside the 'PacketFence' app in Azure AD
+  1. Tenant ID: the 'Tenant ID' that was displayed while configuring the 'PacketFence' app inside Azure
+  1. Add any authentication or administration rules and then save the source
+
+With this configuration, you can now use this source in your connection profiles to authenticate and authorize users on the captive portal and use it with EAP-TLS to authorize users (getting the role and access duration) as long as your EAP-TLS certificates use the distinguished name of the Azure AD users as their common name. Additionally, you can use this source for authenticating users in the admin interface and for VPN access.
+
+==== Using Azure AD in 802.1x
+
+You can perform 802.1x authentication of users using Azure AD but this will only work with supplicants configured to perform EAP-TTLS PAP which provides the RADIUS server with the plain-text password of the user. Support for this type of authentication is not as broad as EAP-PEAP MSCHAPv2 in the 802.1x supplicants but unfortunately Azure AD doesn't support MSCHAP authentication. Refer to the documentation of your operating system on how to configure EAP-TTLS PAP. This section will only focus on enabling EAP-TTLS PAP for your Azure AD users in PacketFence.
+
+ 1. Under 'Configuration->Policies and Access Control->Realms', create a new realm
+  1. Realm: enter the realm of your Azure AD users. Example, if the usernames have the following format `bob@inverseinc.onmicrosoft.com`, then your realm is `inverseinc.onmicrosoft.com`
+  1. Go in the 'Stripping' tab of the realm and select your Azure AD source under 'Azure AD Source for TTLS PAP'
+  1. Still in the 'Stripping' tab, disable (uncheck), the following settings:
+   1. Strip on the portal
+   1. Strip on the admin
+   1. Strip in RADIUS authorization
+  1. Save the realm
```

Comment:
Don't we have to set no strip in the freeradius proxy options ?

Reply:
Not according to my tests, I got the username unstripped in httpd.aaa with that configuration